code/modules/html_interface/map/station_map.dm

```diff
@@ -158,6 +182,12 @@
 										canvas.DrawBox(HOLOMAP_PATH, min(i+map.holomap_offset_x[zLevel],((2 * world.view + 1)*WORLD_ICON_SIZE)), min(r+map.holomap_offset_y[zLevel],((2 * world.view + 1)*WORLD_ICON_SIZE)))
 									else
 										canvas.DrawBox(HOLOMAP_PATH, i, r)
+						else if (map.snow_theme && (zLevel==map.zMainStation) && istype(aera, /area/vault))
```

Comment:
it might be better to make this a var on the z level itself rather than a map var

that way you can cause blurring on any z level of any map you like! (same with the other uses of snow_theme in this file)

Reply:
I would agree but regular Asteroid Z levels and "kraken" Asteroid Z levels use the same /datum/zLevel/mining . Can I replace the subtype in snaxi.dm without breaking something else?

Reply:
I have looked over the code and it looks safe to replace the subtype. No other code references it by type and its definition is just a name (L224 _map.dm)

Reply:
That should do it. After a quick test it appears to work well.